CHANGELOG.rst

```diff
@@ -11,10 +11,77 @@ Automatic Migrations
 --------------------
 - If the ``risk:vuln:cvss:av`` property equals ``V`` it is migrated to ``P``.
   (`#3013 <https://github.com/vertexproject/synapse/pull/3013>`_)
+- Parse ``inet:http:cookie`` nodes to populate the newly added
+  ``:name`` and ``:value`` properties.
+  (`#3015 <https://github.com/vertexproject/synapse/pull/3015>`_)
 - See :ref:`datamigration` for more information about automatic migrations.
 
 Features and Enhancements
 -------------------------
+- Added the ``belief`` model which includes the following new forms:
+  (`#3015 <https://github.com/vertexproject/synapse/pull/3015>`_)
+
+  ``belief:system``
+  A belief system such as an ideology, philosophy, or religion.
+
+  ``belief:tenet``
+  A concrete tenet potentially shared by multiple belief systems.
+
+  ``belief:subscriber``
+  A contact which subscribes to a belief system.
+
+  ``belief:system:type:taxonomy``
+  A hierarchical taxonomy of belief system types.
+
+- Added declaration for ``risk:compromise -(uses)> ou:technique``
+  light-weight edges.
+  (`#3015 <https://github.com/vertexproject/synapse/pull/3015>`_)
+
+- Updated ``inet:http:session`` and ``inet:http:request`` forms to
+  include the following property:
+  (`#3015 <https://github.com/vertexproject/synapse/pull/3015>`_)
+
+  ``:cookies``
+  An array of ``inet:http:cookie`` values associated with the node.
+
+- Updated the ``inet:http:cookie`` form to include the following properties:
```

Comment:
changelog text need to be < 80 characters wide otherwise the slack post from the release announcement will look odd.